src/workerd/api/node/tests/process-stdio-nodejs-test.js

```diff
@@ -11,10 +10,6 @@ export const processStdioPropertiesTest = {
       process.stdin instanceof Readable,
       'stdin should be instance of Readable'
     );
-    assert(
-      process.stdin instanceof ReadStream,
```

Comment:
Why did we remove this test?

Reply:
I updated this test case to test without the FS compat flag, moving this test instead to the new stdio-fs test.